react/facilitator_controls.js

```diff
@@ -43,10 +45,45 @@ class FacilitatorControls extends Component {
   }
 
   async setBuggedEnvelopes() {
+    let selection = []
+    let movedEnvelopes = []
+    for (let i = 0; i < this.state.team2Batch.length; i++) {
+      movedEnvelopes.push(this.state.team2Batch[i]);
+    }
+    movedEnvelopes.push(this.state.selectedTeam1Id);
+    selection.push(this.state.selectedTeam1Id);
+    for (let i = 0; i < this.state.selectedTeam2Id.length; i++) {
+      selection.push(this.state.selectedTeam2Id[i]);
+    }
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({
+          envelopes: selection,
+        })
+      };
+    fetch('/api/set-changed', requestOptions);
```

Comment:
did you mean to use `await` here?  if not, then this function doesn't require the `async` keyword.